client/flags/flags.go

```diff
@@ -123,6 +126,14 @@ func AddTxFlagsToCmd(cmd *cobra.Command) {
 	viper.BindPFlag(FlagKeyringBackend, cmd.Flags().Lookup(FlagKeyringBackend))
 }
 
+// AddPaginationFlagsToCmd adds common paginations flags to command
+func AddPaginationFlagsToCmd(cmd *cobra.Command, query string) {
+	cmd.Flags().String(FlagPageKey, "", fmt.Sprintf("pagination page-key of %s to query for", query))
+	cmd.Flags().Uint64(FlagOffset, 1, fmt.Sprintf("pagination offset of %s to query for", query))
```

Comment:
default should be `0`
```suggestion
	cmd.Flags().Uint64(FlagOffset, 0, fmt.Sprintf("pagination offset of %s to query for", query))
```